src/pages/landing/stocks/_footerBanner.tsx

```diff
@@ -128,19 +126,33 @@ const StyledHeaderSmall = styled(Header)`
     }
 `
 
-const FooterBanner = ({ background_pattern, data, is_ppc, small_title, title }) => {
-    const BackgroundPattern = styled.img`
-        position: absolute;
-        top: -298px;
-        left: -8%;
-        z-index: 1;
-        width: 558px;
+const BackgroundPattern = styled.img`
+    position: absolute;
+    top: -298px;
+    left: -8%;
+    z-index: 1;
+    width: 558px;
 
-        @media ${device.tabletL} {
-            display: none;
-        }
-    `
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
 
+type FooterBannerProps = {
+    background_pattern: string
+    is_ppc: boolean
+    title: ReactElement
+    small_title: ReactElement
+    data: { stocks_banner: any }
```

Comment:
Is stocks_banner an image data? If it is for used for Gatsby Image, you can use the `ImageDataLike` type from `gatsby-plugin-image`.